deploy/samples/api_v1alpha2_astarte_voyager_ingress_cr_minimal.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: api.astarte-platform.org/v1alpha1
+kind: AstarteVoyagerIngress
+metadata:
+  name: astarte-ingress
+  namespace: astarte
+spec:
+  ### Astarte Voyager Ingress CRD
+  astarte: astarte
+  api:
+    deploy: true
+    exposeHousekeeping: true
+  dashboard:
+    ssl: true
+    host: "dashboard.astarte-example.com" # When not specified, dashboard will be deployed in /dashboard in the API host.
+  broker:
+    deploy: true
+  letsencrypt:
+    use: true
+    acmeEmail: info@myowndomain.com
+    challengeProvider:
+      dns:
+        provider: digitalocean
```

Comment:
are we keeping digitalocean in our examples? we might change with something more generic